### PR TITLE
CB-15759 Create test for cluster stop start with external database

### PR DESCRIPTION
**Pull request by lturcsanyi**
Required CM services should be in running state (cluster start command should be issued) during cluster start.

See detailed description in the commit message.

### CODE REVIEW

File: integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/environment/EnvironmentStopStartTests.java [hunk 1]
```diff
@@ -100,8 +114,19 @@ public void testCreateStopStartEnvironment(TestContext testContext) {
                 .given(EnvironmentTestDto.class)
                 .when(environmentTestClient.start())
                 .await(EnvironmentStatus.AVAILABLE)
+                .given("dx1", DistroXTestDto.class)
+                .await(STACK_AVAILABLE, RunningParameter.key("dx1"))
+                .awaitForHealthyInstances()
+                .then(this::verifyCmServicesStartedSuccessfully)
```
Comment (horadla23) [inline integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/environment/EnvironmentStopStartTests.java:120]: do you want to validate this on every provider?

Comment (lturcsanyi) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/environment/EnvironmentStopStartTests.java:120]: Yes

Comment (horadla23) [reply integration-test/src/main/java/com/sequenceiq/it/cloudbreak/testcase/e2e/environment/EnvironmentStopStartTests.java:120]: ok

**Review by horadla23 — APPROVED** [review]

**Comment by biharitomi** [comment]
@lturcsanyi this will affect all the E2E test runs and will increase the time consumption. Should we enable this on non-long-running E2E test cases too?
I am afraid that the number of tests ended up with timeout is going to happen.

**Comment by lturcsanyi** [comment]
> @lturcsanyi this will affect all the E2E test runs and will increase the time consumption. 

I think it is a valid test to wait for the environment and the related datahub to start as well. The CM service checking shouldnt take too much time.

> Should we enable this on non-long-running E2E test cases too?

This EnvironmentStopStartTests is running only in long-running testcases, or am I wrong?

**Review by biharitomi — APPROVED** [review]

**Pull request merged**